src/Editor/README.md

```diff
@@ -0,0 +1,17 @@
+# Editor
+It provides convenience to users who create cfg files using vscode (text editor).
+- CodeLens
+  - First, a codelens will be provided at the top of each tool.
+  - If you click on the corresponding codelens, you will be provided with an additional codelens for the sub-attributes.
+  - If you press the codelens for each attribute, that codelens disappears.
+  - If you want to see the hidden property again, press the tool's codelens.
+- Hover
+  - This shows a list of options if you mouse over the names of each tool.
+
+## Demo

```

Comment:
How about move the gif just below each items `CodeLens` and `Hover` and remove `Demo` ?
VSCode release notes explains like this.

Reply:
Is it correct that I revised it as you requested?

Reply:
Yup :)